frameworks/hdfs/universe/config.json

```diff
@@ -164,6 +164,11 @@
       "type": "object",
       "description": "HDFS File System configuration options",
       "properties": {
+        "version": {
```

Comment:
Please don't expose this as configurable.  We don't allow the changing of the underlying service version within the context of a single package version.